index.html

```diff
@@ -62,6 +62,177 @@ <h1 class="header-title">I’m Abdallah Glad to see you!</h1>
+            </svg>
+            </a>
+           </li>    
+        </ul>
+        <button class="p-btn">Get my resume</button>
+        <ul class="tech-list">
+          <li class="unclicked-tech-list-f tech-list-f">
+            <h4>Languages</h4>
+            <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path fill-rule="evenodd" clip-rule="evenodd" d="M5.29289 8.29289C5.68342 7.90237 6.31658 7.90237 6.70711 8.29289L12 13.5858L17.2929 8.29289C17.6834 7.90237 18.3166 7.90237 18.7071 8.29289C19.0976 8.68342 19.0976 9.31658 18.7071 9.70711L12.7071 15.7071C12.3166 16.0976 11.6834 16.0976 11.2929 15.7071L5.29289 9.70711C4.90237 9.31658 4.90237 8.68342 5.29289 8.29289Z" fill="#5E6C84"/>
+            </svg>
+                
+        </li>
+           <ul class="language-list">
+            <li class="unclicked-tech-list-p tech-list-f">
+                <img  src="images/js.png" alt="portfolio project" width="35"
+                height="35" />
+                <span class="ln-text">javascript</span>
+            </li>
+            <li class="tech-list-f">
+                <img src="images/html.png" alt="portfolio project" width="35"
+                height="35" />
+                <span class="ln-text">HTML</span>
+            </li>
+            <li class="tech-list-f">
+                <img src="images/css.png" alt="portfolio project" width="35"
+                height="35" />
+                <span class="ln-text">CSS</span>
+            </li>
+          </ul>
+          <li class="unclicked-tech-list unclicked-tech-list-p  tech-list-f">
+            <h4>Framework</h4> 
+            <svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path fill-rule="evenodd" clip-rule="evenodd" d="M9.29289 5.43962C9.68342 5.0491 10.3166 5.0491 10.7071 5.43962L16.7071 11.4396C17.0976 11.8301 17.0976 12.4633 16.7071 12.8538L10.7071 18.8538C10.3166 19.2444 9.68342 19.2444 9.29289 18.8538C8.90237 18.4633 8.90237 17.8301 9.29289 17.4396L14.5858 12.1467L9.29289 6.85384C8.90237 6.46331 8.90237 5.83015 9.29289 5.43962Z" fill="#505F79"/>
+            </svg>
+             </li>
+          <li class="unclicked-tech-list unclicked-tech-list-p  tech-list-f">
+            <h4>Skills</h4>
+            <svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path fill-rule="evenodd" clip-rule="evenodd" d="M9.29289 5.43962C9.68342 5.0491 10.3166 5.0491 10.7071 5.43962L16.7071 11.4396C17.0976 11.8301 17.0976 12.4633 16.7071 12.8538L10.7071 18.8538C10.3166 19.2444 9.68342 19.2444 9.29289 18.8538C8.90237 18.4633 8.90237 17.8301 9.29289 17.4396L14.5858 12.1467L9.29289 6.85384C8.90237 6.46331 8.90237 5.83015 9.29289 5.43962Z" fill="#505F79"/>
+            </svg>
+           </li>
+        </ul> 
+    </section>
```

Comment:
- [ ] Kindly make few adjustments to make this section look more like the one on Figma. This includes `background-color`, `font-size` and the `width` of the sublist of the languages 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/192098694-b6a678b5-f055-43c9-aa5f-b70bde530018.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/192098701-638b4577-6e44-4997-8440-50a571d09f4e.png)

Reply:
Hi @tufoinnkuo10   I have to try to resolve all the issues but I still got stuck on one thing, the interactive button I think the interactive button means something like mouse hover, button pressed, etc . But when I looking my Figma design  templates don't see those staff just because I'm to Figma please can you send to me a title guide on how to get those interaction properties on Figma